### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by weiwensangsang**
First, I think the Readme tells me that the maven and gradle are the two ways to use Guava,  I am a little confuse about what "xml" means. I think it means "pom.xml". But "maven" is clearer.
Next, I think it would be clearer to describe the two situations **separately**. I introduce this project to my students and invite them to use Guava in their future work. However, in the early days of their learning, changing google's code was **more difficult** than copying google's code. I hope to lower the learning threshold of this project. This is also the purpose I have been working on.

### CODE REVIEW

**Comment by jbduncan** [comment]
As someone not part of the Guava team, this looks good to me!

@weiwensangsang Just to warn you that it takes a long time for the Guava team to look at issues and review PRs, so expect to wait a month or so before you get a response.

**Comment by weiwensangsang** [comment]
> As someone not part of the Guava team, this looks good to me!
> 
> @weiwensangsang Just to warn you that it takes a long time for the Guava team to look at issues and review PRs, so expect to wait a month or so before you get a response.

so sad to hear that.

File: README.md [hunk 1]
```diff
@@ -26,12 +26,18 @@ Guava, see [using Guava in your build].
 
 To add a dependency on Guava using Maven, use the following:
 
-```xml
+```maven
```
Comment (liach) [inline README.md]:
```suggestion
```xml
```
Your change broke syntax highlighting.

File: README.md [hunk 2]
```diff
@@ -26,12 +26,18 @@ Guava, see [using Guava in your build].
...
 <dependency>
   <groupId>com.google.guava</groupId>
   <artifactId>guava</artifactId>
   <version>28.1-jre</version>
-  <!-- or, for Android: -->
+</dependency>
+```
+or, for Android:
+```
```
Comment (liach) [inline README.md]:
```suggestion
```xml
```

File: README.md [hunk 3]
```diff
@@ -41,7 +47,11 @@ To add a dependency using Gradle:
 ```gradle
 dependencies {
   compile 'com.google.guava:guava:28.1-jre'
-  // or, for Android:
+}
+```
+or, for Android:
+```
```
Comment (liach) [inline README.md]:
```suggestion
```gradle
```

**Pull request closed without merging**